var/spack/repos/builtin/packages/jbigkit/package.py

```diff
@@ -0,0 +1,53 @@
+##############################################################################
+# Copyright (c) 2013-2017, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Jbigkit(MakefilePackage):
+    """JBIG-Kit is a software implementation of the JBIG1 data compression standard."""
+
+    homepage = "http://www.cl.cam.ac.uk/~mgk25/jbigkit/"
+    url      = "http://www.cl.cam.ac.uk/~mgk25/jbigkit/download/jbigkit-2.1.tar.gz"
+
+    version('2.1', 'ebcf09bed9f14d7fa188d3bd57349522')
+    version('2.0', '3dd87f605abb1a97a22dc79d8b3e8f6c')
+    version('1.6', 'ce196e45f293d40ba76af3dc981ccfd7')
+
+    def edit(self, spec, prefix):
+        makefile = FileFilter('libjbig/Makefile')
+        makefile.filter('CC = .*', 'CC = cc')
+
+    def install(self, spec, prefix):
+        with working_dir('libjbig'):
+            make()
```

Comment:
If you check out `MakefilePackage`, you'll see that it has 3 phases, "edit", "build", and "install". The "build" phase already runs `make()` for you, so you would be running `make` twice here. If you add:
```python
build_directory = 'libjbig'
```
to the package, you can get rid of this call to `make`. The rest of the install method can remain the same, although I would replace:
```python
with working_dir('libjbig'):
```
with:
```python
with working_dir(self.build_directory):
```
to keep things consistent.

Reply:
Fixed.